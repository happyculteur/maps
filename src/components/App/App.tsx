import * as React from "react";
import { AppBar } from "../AppBar";

class App extends React.Component {
  public render() {
    return (
      <div>
        <AppBar />
      </div>
    );
  }
}

export default App;
