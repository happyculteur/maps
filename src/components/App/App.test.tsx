import { shallow } from "enzyme";
import * as React from "react";
import App from "./App";

it("should renders without crashing", () => {
  const wrapper = shallow(<App />);

  expect(wrapper.debug()).toMatchSnapshot();
});
