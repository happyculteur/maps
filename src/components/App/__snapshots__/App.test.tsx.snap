// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders without crashing 1`] = `
"<ThemeProvider theme={{...}}>
  <WithStyles(CssBaseline) />
  <div className=\\"application\\">
    <HelmetWrapper defer={true} encodeSpecialCharacters={true}>
      <meta charSet=\\"utf-8\\" />
      <meta name=\\"viewport\\" content=\\"width=device-width, initial-scale=1, shrink-to-fit=no\\" />
      <meta name=\\"theme-color\\" content=\\"#000000\\" />
      <link rel=\\"manifest\\" href=\\"/manifest.json\\" />
      <link rel=\\"shortcut icon\\" href=\\"/favicon.ico\\" />
      <link rel=\\"stylesheet\\" href=\\"https://fonts.googleapis.com/css?family=Roboto:300,400,500\\" />
      <link rel=\\"stylesheet\\" href=\\"https://fonts.googleapis.com/icon?family=Material+Icons\\" />
      <title>
        Happyculteur Maps
      </title>
    </HelmetWrapper>
    <AppBar />
  </div>
</ThemeProvider>"
`;
